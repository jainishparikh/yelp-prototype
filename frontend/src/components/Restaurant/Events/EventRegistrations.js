import React, { Component } from 'react'
import { Link } from 'react-router-dom';
import axios from 'axios';
import BACKEND_URL from '../../../config/config'

export class EventRegistrations extends Component {
    constructor( props ) {
        super( props )
        this.state = {
            Attendees: [],
            eventID: this.props.eventData.eventID,
        }
    }
    componentDidMount () {
        var eventID = this.state.eventID
        axios.get( BACKEND_URL + '/events/attendees/' + eventID ).then( response => {
            response.data.map( ( attendee ) => {
                this.setState( {
                    Attendees: [ ...this.state.Attendees, attendee ]
                } )

            } )
            console.log( this.state )

        } ).catch( error => {
            console.log( "Error getting attendees for an event" )
        } )
    }
    render () {
        let details = this.state.Attendees.map( ( attendee ) => {

            return (
                <tr>
                    <td style={ { "padding": "10px", "border": "1px solid gray" } } >{ attendee.userName }</td>
                    <td style={ { "padding": "10px", "border": "1px solid gray" } }>{ attendee.userEmail }</td>
                    <td style={ { "padding": "10px", "border": "1px solid gray" } }>
                        <Link className="btn btn-primary" to={ `/restaurants/userprofiles/${ attendee.userEmail }/${ attendee.userID }` } >
                            View Profile
                    </Link>
                    </td>
                </tr>
            )
        } )
        return (
            <div>
                <div className="container">
                    <div className="details">
                        <h2>List of Registered People:</h2>
                        <table>
                            <tbody>

                                { details }
                            </tbody>
                        </table>

                    </div>
                    <div className="row mt-5">
                        <div className="col-3">
                            <button className="btn btn-danger" onClick={ this.props.closePopUp }>Back</button>
                        </div>

                    </div>
                </div>

            </div>
        )
    }
}

export default EventRegistrations
