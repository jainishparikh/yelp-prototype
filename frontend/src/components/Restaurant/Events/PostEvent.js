import React, { Component } from 'react'
import { Redirect } from 'react-router';
//import { Link } from 'react-router-dom';
import cookie from 'react-cookies';
import axios from 'axios';
import BACKEND_URL from '../../../config/config'

export class PostEvent extends Component {
    constructor( props ) {
        super( props )
        this.state = {
            eventID: "",
            eventName: "",
            eventTime: "",
            eventDate: "",
            eventDescription: "",
            eventLocation: "",
            hashtags: "",
        }
    }

    //handle input change
    handleInputChange = inp => {
        // console.log( inp.target.name, inp.target.value );
        this.setState( {
            [ inp.target.name ]: inp.target.value
        } )
    }

    handleOnSubmit = sub => {
        sub.preventDefault();
        var eventData = {
            eventName: this.state.eventName,
            eventTime: this.state.eventTime,
            eventDate: this.state.eventDate,
            eventDescription: this.state.eventDescription,
            eventLocation: this.state.eventLocation,
            Hashtags: this.state.hashtags,
            restaurantID: cookie.load( 'id' ),
        }
        axios.post( BACKEND_URL + "/events/restaurants/addEvent", eventData ).then( response => {
            if ( response.status === 200 ) {
                console.log( "Event successfully Posted" + response );

                window.location.assign( "/restaurants/events" )
            }
        } ).catch( err => {
            console.log( "Error in Posting event" );
        } )
    }

    render () {
        var redirectVar = null;
        if ( !cookie.load( "auth" ) ) {
            redirectVar = <Redirect to="/login" />
        }
        return (
            <div>
                { redirectVar }
                <div className="container" >

                    <form onSubmit={ this.handleOnSubmit }>
                        <div className="row mt-2">
                            <h2>Provide Event details below:</h2>

                        </div>
                        <div className="row mt-2">
                            <div className="col-3">
                                Name: <input type="text" className="form-control" name="eventName" required autoFocus
                                    placeholder={ this.state.eventName } onChange={ this.handleInputChange } />
                            </div>
                            <div className="col-3">
                                Hashtags: <input type="text" className="form-control" name="hashtags" required
                                    placeholder={ this.state.Hashtags } onChange={ this.handleInputChange } />
                            </div>

                        </div>
                        <div className="row">
                            <div className="col-3">
                                Time:<input type="text" className="form-control" name="eventTime" required
                                    placeholder={ this.state.eventTime } onChange={ this.handleInputChange } />
                            </div>
                            <div className="col-3">
                                Date:<input type="date" className="form-control" name="eventDate" required
                                    placeholder={ this.state.eventDate } onChange={ this.handleInputChange } />
                            </div>
                        </div>
                        <div className="row">
                            <div className="col-6">
                                Location: <input type="text" className="form-control" name="eventLocation" required
                                    placeholder={ this.state.eventLocation } onChange={ this.handleInputChange } />
                            </div>
                        </div>
                        <div className="row">
                            <div className="col-6">
                                Description: <input type="text" className="form-control" name="eventDescription" required
                                    placeholder={ this.state.eventDescription } onChange={ this.handleInputChange } />
                            </div>
                        </div>

                        <div className="row mt-5">
                            <div className="col-3">
                                <button type="submit" className="btn btn-primary">POST</button>
                            </div>
                            <div className="col-3">
                                <button className="btn btn-danger" onClick={ this.props.closePopUp }>Cancel</button>
                            </div>

                        </div>
                    </form>
                </div>


            </div>
        )
    }
}

export default PostEvent
