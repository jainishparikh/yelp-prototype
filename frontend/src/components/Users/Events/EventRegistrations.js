import React, { Component } from 'react'
import cookie from 'react-cookies';
import axios from 'axios';
import BACKEND_URL from '../../../config/config'
import { connect } from "react-redux";

export class EventRegistrations extends Component {
    constructor( props ) {
        super( props )
        this.state = {
            eventID: this.props.eventData._id,
            eventName: this.props.eventData.eventName,
            eventDescription: this.props.eventData.eventDescription,
            eventTime: this.props.eventData.eventTime,
            eventDate: this.props.eventData.eventDate,
            eventLocation: this.props.eventData.eventLocation,
            Hashtags: this.props.eventData.Hashtags,
            restaurantID: this.props.eventData.restaurantID,
            registered: false
        }
    }
    componentDidMount () {
        let exist = this.props.RegisteredEvents.some( event => event._id === this.state.eventID )
        if ( exist ) {
            this.setState( {
                registered: true
            } )
        }

    }
    handleRegistration = ( e ) => {
        var data = {
            userID: cookie.load( 'id' ),
            eventID: this.state.eventID,
            userName: cookie.load( 'name' ),
            userEmail: cookie.load( 'email' )
        }
        axios.post( BACKEND_URL + '/events/users/register', data ).then( response => {
            console.log( "Registration Successfull" );
            window.location.assign( '/users/events' )
        } ).catch( error => {
            console.log( "Error in user registration: ", error )
        } )

    }

    render () {
        var register = null
        if ( this.state.registered ) {
            register = <button className="btn btn-primary" >Already Registered</button>
        } else {
            register = <button className="btn btn-primary" onClick={ this.handleRegistration }>Register</button>
        }
        return (
            <div>
                <div className="container">
                    <div className="row">
                        <div className="col-4"><h6>Name:</h6> { this.props.eventData.eventName }</div>
                        <div className="col-4"><h6>Hashtags:</h6> { this.props.eventData.Hashtags }</div>

                    </div>
                    <div className="row">
                        <div className="col-4"><h6>When:</h6> { this.props.eventData.eventTime }, { this.props.eventData.eventDate }</div>
                        <div className="col-4"></div>

                    </div>
                    <div className="row">
                        <div className="col-4"><h6>Where:</h6> { this.props.eventData.eventLocation }</div>

                    </div>
                    <div className="row">
                        <div className="col-8"><h6>Description:</h6> { this.props.eventData.eventDescription }</div>

                    </div>


                    <div className="row mt-5">
                        <div className="col-3">
                            { register }
                        </div>

                        <div className="col-3">
                            <button className="btn btn-danger" onClick={ this.props.closePopUp }>Back</button>
                        </div>

                    </div>
                </div>

            </div>
        )
    }
}

const matchStateToProps = ( state ) => {
    return {
        Events: state.eventsReducer.Events,
        RegisteredEvents: state.eventsReducer.RegisteredEvents,
    }

}

export default connect( matchStateToProps )( EventRegistrations )

