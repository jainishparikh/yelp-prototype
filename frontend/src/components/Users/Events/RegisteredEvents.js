import React, { Component } from 'react'

export class RegisteredEvents extends Component {
    render () {
        return (
            <div>
                <div className="container mt-2 pt-2 pb-2" style={ { "background": "whitesmoke" } } >
                    <div className="row">

                        <div className="col-6">
                            <h4>{ this.props.eventData.eventName }</h4>
                        </div>
                        <div className="col-6">
                            { this.props.eventData.eventTime }
                        </div>
                    </div>
                    <div className="row">
                        <div className="col-6">
                            { this.props.eventData.Hashtags }
                        </div>
                        <div className="col-6">
                            { this.props.eventData.eventDate }
                        </div>

                    </div>
                    <div className="row">
                        <div className="col-12">
                            { this.props.eventData.eventDescription }
                        </div>

                    </div>

                </div>

            </div>
        )
    }
}

export default RegisteredEvents
