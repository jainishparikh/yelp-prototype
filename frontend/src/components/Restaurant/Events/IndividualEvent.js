import React, { Component } from 'react'
import Modal from 'react-modal';
import EventRegistrations from './EventRegistrations';

export class IndividualEvent extends Component {
    constructor( props ) {
        super( props );
        this.state = {
            eventID: this.props.eventData._id,
            eventName: this.props.eventData.eventName,
            eventDescription: this.props.eventData.eventDescription,
            eventTime: this.props.eventData.eventTime,
            eventDate: this.props.eventData.eventDate,
            eventLocation: this.props.eventData.eventLocation,
            Hashtags: this.props.eventData.Hashtags,
            restaurantID: this.props.eventData.ref_restaurantID,
            registartionsPopUp: false,
        }
    }

    toggleRegistrationsPopUp = ( e ) => {
        this.setState( {
            registartionsPopUp: !this.state.registartionsPopUp
        } )
    }

    render () {
        return (
            <div style={ { "width": "90%" } }>
                <div className="container pt-5" >
                    <div className="row" style={ { "padding": "10px", "background": "whitesmoke", "box-shadow": "0px 0px 10px gray" } }>
                        <div className="col-9">
                            <div className="row">

                                <div className="col-8">
                                    <h2>{ this.state.eventName }</h2>
                                </div>
                                <div className="col-4">
                                    <h6>Time:</h6>
                                    { this.state.eventTime }
                                </div>
                            </div>
                            <div className="row">
                                <div className="col-8" style={ { "color": "blue" } }>
                                    { this.state.Hashtags }
                                </div>
                                <div className="col-4">
                                    <h6>Date:</h6>
                                    { this.state.eventDate }
                                </div>

                            </div>
                            <div className="row">
                                <div className="col-8">
                                    { this.state.eventDescription }
                                </div>
                                <div className="col-4"></div>

                            </div>
                        </div>
                        <div className="col-3">
                            <div className="post-event m-2" >
                                <button className="btn btn-danger" onClick={ this.toggleRegistrationsPopUp }>View Registrations</button>
                            </div>
                            {/* using react-modal for popup to add dish */ }
                            <Modal isOpen={ this.state.registartionsPopUp } style={ {
                                overlay: {
                                    position: 'fixed',
                                    top: 0,
                                    left: 0,
                                    right: 0,
                                    bottom: 0,
                                    backgroundColor: 'rgba(255, 255, 255, 0.25)'
                                },
                                content: {
                                    position: 'relative',
                                    top: '20%',
                                    left: '20%',
                                    right: '20%',
                                    bottom: '20%',
                                    border: '2px solid #ccc',
                                    background: '#fff',
                                    overflow: 'auto',
                                    WebkitOverflowScrolling: 'touch',
                                    borderRadius: '4px',
                                    outline: 'none',
                                    padding: '20px'
                                }
                            } } >
                                <EventRegistrations eventData={ this.state } closePopUp={ this.toggleRegistrationsPopUp } />
                            </Modal>

                        </div>

                    </div>






                </div>
            </div>
        )
    }
}

export default IndividualEvent
