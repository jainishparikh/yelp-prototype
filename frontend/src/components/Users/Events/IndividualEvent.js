import React, { Component } from 'react'
import Modal from 'react-modal';
import EventRegistrations from './EventRegistrations';

export class IndividualEvent extends Component {
    constructor( props ) {
        super( props );
        this.state = {
            eventID: this.props.eventData._id,
            eventName: this.props.eventData.eventName,
            eventDescription: this.props.eventData.eventDescription,
            eventTime: this.props.eventData.eventTime,
            eventDate: this.props.eventData.eventDate,
            eventLocation: this.props.eventData.eventLocation,
            Hashtags: this.props.eventData.Hashtags,
            restaurantID: this.props.eventData.restaurantID,
            registrationsPopUp: false,
        }
    }

    toggleRegistrationsPopUp = ( e ) => {
        this.setState( {
            registrationsPopUp: !this.state.registrationsPopUp
        } )
    }


    render () {

        return (
            <div>
                <div className="container pt-2 pb-2" >
                    <div className="row" style={ { "background": "whitesmoke" } }>
                        <div className="col-10">
                            <div className="row">

                                <div className="col-8">
                                    <h2>{ this.props.eventData.eventName }</h2>
                                </div>
                                <div className="col-4">
                                    <h6>Time:</h6>
                                    { this.props.eventData.eventTime }
                                </div>
                            </div>
                            <div className="row">
                                <div className="col-8" style={ { "color": "blue" } }>
                                    { this.props.eventData.Hashtags }
                                </div>
                                <div className="col-4">
                                    <h6>Date:</h6>
                                    { this.props.eventData.eventDate }
                                </div>

                            </div>
                            <div className="row">
                                <div className="col-8">
                                    { this.props.eventData.eventDescription }
                                </div>
                                <div className="col-4"></div>

                            </div>
                        </div>
                        <div className="col-2">
                            <div className="post-event m-2" >
                                <button className="btn btn-danger" onClick={ this.toggleRegistrationsPopUp }>Register</button>
                            </div>
                            {/* using react-modal for popup to add dish */ }
                            <Modal isOpen={ this.state.registrationsPopUp } style={ {
                                overlay: {
                                    position: 'fixed',
                                    top: 0,
                                    left: 0,
                                    right: 0,
                                    bottom: 0,
                                    backgroundColor: 'rgba(255, 255, 255, 0.25)'
                                },
                                content: {
                                    position: 'relative',
                                    top: '20%',
                                    left: '20%',
                                    right: '20%',
                                    bottom: '20%',
                                    border: '2px solid #ccc',
                                    background: '#fff',
                                    overflow: 'auto',
                                    WebkitOverflowScrolling: 'touch',
                                    borderRadius: '4px',
                                    outline: 'none',
                                    padding: '20px'
                                }
                            } } >
                                <EventRegistrations eventData={ this.props.eventData } closePopUp={ this.toggleRegistrationsPopUp } />
                            </Modal>

                        </div>

                    </div>

                </div>
            </div>
        )
    }
}
export default IndividualEvent