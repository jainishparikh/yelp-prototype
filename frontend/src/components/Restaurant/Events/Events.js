import React, { Component } from 'react'
import { Redirect } from 'react-router';
import cookie from 'react-cookies';
import axios from 'axios';
import IndividualEvent from './IndividualEvent';
import BACKEND_URL from '../../../config/config';
import Modal from 'react-modal';
import PostEvent from './PostEvent';
export class Events extends Component {
    constructor( props ) {
        super( props )
        this.state = {
            Events: [],
            postEventPopUp: false
        }
    }

    componentDidMount () {
        var restaurantID = cookie.load( 'id' )
        // var restaurantID = 13
        return axios.get( BACKEND_URL + '/events/restaurants/' + restaurantID ).then( response => {
            // response.data.map( ( event ) => {
            //     this.setState( {
            //         Events: [ ...this.state.Events, event ]
            //     } )

            // } )
            this.setState( {
                Events: response.data
            } )
            // console.log( this.state )

        } ).catch( error => {
            console.log( "Error in fetching restaurant events: ", error );
        } )
    }

    togglepostEventPopUp = ( e ) => {
        this.setState( {
            postEventPopUp: !this.state.postEventPopUp
        } )
    }


    render () {
        var redirectVar = null;
        if ( !( cookie.load( "auth" ) && cookie.load( "type" ) === "restaurants" ) ) {
            redirectVar = <Redirect to="/login" />
        }
        let details = this.state.Events.map( ( event, index ) => {
            return (
                <IndividualEvent key={ index } eventData={ event } />
            )
        } )
        return (
            <div>
                { redirectVar }
                <div className="container-fluid m-1" style={ { height: "100vh" } }>
                    <div className="row mt-10" >
                        <div className='col-2'>
                            <div className="row mt-2" style={ { height: "240px" } }> <h2> Events Posted by You </h2></div>
                            <div className="row mt-2" style={ { height: "520px" } }>
                                <div className="post-event m-2" >
                                    <button className="btn btn-danger" onClick={ this.togglepostEventPopUp }>Post Event</button>
                                </div>
                                {/* using react-modal for popup to add dish */ }
                                <Modal isOpen={ this.state.postEventPopUp } style={ {
                                    overlay: {
                                        position: 'fixed',
                                        top: 0,
                                        left: 0,
                                        right: 0,
                                        bottom: 0,
                                        backgroundColor: 'rgba(255, 255, 255, 0.25)'
                                    },
                                    content: {
                                        position: 'relative',
                                        top: '20%',
                                        left: '20%',
                                        right: '20%',
                                        bottom: '20%',
                                        border: '2px solid #ccc',
                                        background: '#fff',
                                        overflow: 'auto',
                                        WebkitOverflowScrolling: 'touch',
                                        borderRadius: '4px',
                                        outline: 'none',
                                        padding: '20px'
                                    }
                                } } >
                                    <PostEvent closePopUp={ this.togglepostEventPopUp } />
                                </Modal>

                            </div>
                        </div>
                        <div className='col-10' style={ { height: "20%" } }>{ details }</div>

                    </div>
                </div>

            </div>
        )
    }
}

export default Events
