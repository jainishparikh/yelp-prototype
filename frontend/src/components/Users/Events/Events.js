import React, { Component } from 'react'
import { Redirect } from 'react-router';
import cookie from 'react-cookies';
import axios from 'axios';
import IndividualEvent from './IndividualEvent';
import BACKEND_URL from '../../../config/config';
import Modal from 'react-modal';
import RegisteredEvents from './RegisteredEvents';

export class Events extends Component {
    constructor( props ) {
        super( props )
        this.state = {
            Events: [],
            RegisteredEvents: [],
            searchInput: ""

        }
    }

    componentDidMount () {
        // getting all events
        axios.get( BACKEND_URL + '/events/' ).then( response => {
            console.log( "got events", response.data )
            response.data.map( ( event ) => {
                this.setState( {
                    Events: [ ...this.state.Events, event ],
                } )

            } )
            console.log( this.state )

        } ).catch( error => {
            console.log( "Error in fetching restaurant events: ", error );
        } )

        //getting registered events
        var id = cookie.load( 'id' )
        axios.get( BACKEND_URL + '/events/users/' + id ).then( response => {
            console.log( "Got registered events" );
            response.data.map( ( event ) => {
                this.setState( {
                    RegisteredEvents: [ ...this.state.RegisteredEvents, event ]
                } )
            } )


        } ).catch( error => {
            console.log( "Error in getting registered events: ", error )
        } )
    }


    handleSearch = ( e ) => {
        console.log( e.target.name, e.target.value )
        this.setState( {
            [ e.target.name ]: e.target.value
        } )
    }

    render () {
        var redirectVar = null;
        if ( !( cookie.load( "auth" ) && cookie.load( "type" ) === "users" ) ) {
            redirectVar = <Redirect to="/login" />
        }
        let filteredEvents = this.state.Events.filter( ( event ) => {
            return event.eventName.toLowerCase().includes( this.state.searchInput.toLowerCase() )
        } )
        let sortedEvents = filteredEvents.sort( ( a, b ) => {
            let d1 = a.eventDate.split( "-" )
            let date1 = d1[ 0 ] + d1[ 1 ] + d1[ 2 ]
            let d2 = b.eventDate.split( "-" )
            let date2 = d2[ 0 ] + d2[ 1 ] + d2[ 2 ]
            return date1 - date2
        } )
        let details = sortedEvents.map( event => {

            return (
                <IndividualEvent eventData={ event } />
            )
        } )

        let registeredEvents = this.state.RegisteredEvents.map( event => {
            return ( <RegisteredEvents eventData={ event } />
            )
        } )

        return (
            <div>
                { redirectVar }
                <div className="container-fluid m-1" style={ { height: "100vh" } }>
                    <div className="row">
                        <div className="col-8">
                            <div className="row mt-2">
                                <div className="col-4"></div>
                                <div className="col-4">
                                    <input type="text" size="50" name="searchInput" onChange={ this.handleSearch } placeholder="Search Event by Name"></input>
                                </div>
                                <div className="col-4"></div>

                            </div>
                            <div className="row m-2" >
                                <div className="col-1"></div>
                                <div className="col-11">
                                    <div className='details'  >{ details }</div>
                                </div>

                            </div>
                        </div>
                        <div className="col-4" style={ { "padding": "0 15px", "border-left": "1px solid #e6e6e6" } }>
                            <div className="row" style={ { "padding": "0 15px" } }><h3>Registered Events</h3></div>
                            <div className="row" style={ { "padding": "0 15px" } }>
                                { registeredEvents }
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        )
    }
}


export default Events
