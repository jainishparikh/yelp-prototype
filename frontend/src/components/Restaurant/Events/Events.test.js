import React from 'react';
import { shallow } from 'enzyme';

import Events from './Events'

describe( 'Events', () => {

    let axios = require( "axios" );
    let MockAdapter = require( "axios-mock-adapter" );
    let mock = new MockAdapter( axios );

    const eventsData = [
        {
            eventID: 8,
            eventName: "Food Delight",
            eventTime: "9:00 AM to 12:00 PM",
            eventDate: "2020-10-21",
            eventDescription: "Try Something New",
            eventLocation: "San Fransisco, CA",
            hashtags: "#Food#Fiesta",

        },
        {
            eventID: 11,
            eventName: "Food Fair",
            eventTime: "8:00 AM to 12:00 PM",
            eventDate: "2020-11-10",
            eventDescription: "Biggest Food Fair",
            eventLocation: "San Jose",
            hashtags: "#Food#Fair",

        }
    ]


    mock.onGet( "http://localhost:3001/events/restaurants/13" ).reply( 200,
        eventsData
    );


    let component

    beforeEach( () => {
        component = shallow( <Events debug /> );
    } );

    it( 'should render correctly in "debug" mode', () => {
        component.instance()
        expect( component ).toMatchSnapshot();
    } );

    it( 'check if all events by  a particular restaurant are received', async ( done ) => {
        component
            .instance()
            .componentDidMount()
            .then( () => {
                // console.log( component.state().Events )
                expect( component.state().Events ).toEqual( eventsData )
                done()
            } )
        expect( component ).toMatchSnapshot();
    } );

} )