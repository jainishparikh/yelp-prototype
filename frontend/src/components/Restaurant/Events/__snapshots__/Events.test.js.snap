// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Events check if all events by  a particular restaurant are received 1`] = `ShallowWrapper {}`;

exports[`Events should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
